### ISSUE TITLE
--spatial-normalize

### ISSUE THREAD

**Issue by guangmingw** [anuragranj/cc#11]
Could I get to know that if you use the `--spatial-normalize `? How should I use it and set the hyperparameters such as `-s`? I added the `--spatial-normalize` to train the depth and pose networks, but the results are very poor. 
However, I found that other paper used it and had a performance improvement.
I found the method `--spatial-normalize` comes from this paper. Thank you.

> C. Wang, J. M. Buenaposada, R. Zhu and S. Lucey, “Learning Depth from Monocular Videos using Direct Methods,” in Conference on Computer Vision and Pattern Recognition, 2018, pp. 2022-2030.

**Comment by anuragranj** [comment]
That's correct. We also found the same effect. For some reason, the spatial-normalize gives us worse results too. In the end, we did not use it. We also borrowed the loss from Wang et al. My implementation looks consistent with theirs. But, at this point, I do not have any intuition on why it doesn't work with Competitive Collaboration.
